kbcstorage/workspaces.py

```diff
@@ -143,3 +144,26 @@ def load_tables(self, workspace_id, table_mapping, preserve=None):
         url = '{}/{}/load'.format(self.base_url, workspace_id)
 
         return self._post(url, data=body)
+
+    # only supports abs workspace
+    # the workspace must be the object returned by the create method otherwise it won't have the connectionString
+    def load_files(self, workspace, file_mapping, preserve=None):
+        if (workspace['type'] != 'file' and workspace['connection']['backend'] != 'abs'):
```

Comment:
i find it weird that the load_tables takes workspace_id and load_files takes workspace (with some "arbitrary" structure) do we need this check at all?

Reply:
omg, no, that was left from dev, I'll switch it back to id, thanks!